### ISSUE TITLE
Upgrade to collection2-core?

### ISSUE THREAD

**Issue by hashcutdev** [ARKHAM-Enterprises/meteor-comments-ui#124]
I noticed that your package depends on aldeed:collection2 and aldeed:simple-schema.

According to the collection2 github readme (https://github.com/aldeed/meteor-collection2), collection2 and simpleSchema are no longer supported. Instead, we're supposed to depend on collection2-core, and then install simpleSchema using npm instead of the meteor package.

There are some syntax changes with the new version of simpleSchema, but is it possible to upgrade to the new packages?

**Comment by matteodem** [comment]
Yeah sure, feel free to add a PR as this should be straight-forward.

**Comment by hashcutdev** [comment]
Ok. Sounds good. I haven't worked on meteor plugins before, but I'll try it and send you a pull request.
